### DISCUSSION TITLE
nRF52840 dongle as coordinator?

### DISCUSSION THREAD

**Discussion by Calimerorulez** [Koenkk/zigbee2mqtt#13350] [Devices]
Hi,

Maybe asked a couple of a hundred times, but I simply can't find the answer.

Is the nRF52840 dongle usable as Zigbee2MQTT coordinator?

https://www.nordicsemi.com/Products/Development-hardware/nrf52840-dongle

**Comment by sjorge** [comment]
To my knowledge, it is not.
